addons/account/models/account_move.py

```diff
@@ -3076,6 +3076,9 @@ def generate_qr_code(self):
 
         if not self.is_invoice():
             raise UserError(_("QR-codes can only be generated for invoice entries."))
+        if self.move_type in ('out_refund', 'in_refund'):
+            # Do not generate QR codes for credit notes
+            return None
```

Comment:
With this fix the previous condition doesn't make much sense anymore
we could have just one check like
```
if self.move_type != 'out_invoice':
    return None
```